### DISCUSSION TITLE
Driver Configuration With Appium 2.0

### DISCUSSION THREAD

**Discussion by ju1ianj0rdan** [appium/appium#16645] [Q&A] [answered]
<img width="1010" alt="Screen Shot 2022-03-23 at 7 22 44 PM" src="https://user-images.githubusercontent.com/53064370/159812415-f295eaba-0fb0-4837-9bf0-07e21b285a98.png">
I currently have this set up with my Appium project as a custom webdriver, but keep having issues with the Driver, specifically where it is saying:

"Could not instantiate custom webdriver provider of type myprovider"

Specifically looking at lines 42 - 45 where I return a driver to be used for my iOS and Android app, is this up to date? This historaclly was IOSDriver<WebElement>..... I've also tried using RemoteDriver, but that gives the same issue. I am just trying to understand what driver I can return that will work.

**Comment by jlipps** [comment, marked as answer]
You should probably raise this at the java-client repo, as it is related to the Java client and not Appium per se.
